Assets/ScriptableRenderLoop/HDRenderLoop/HDRenderLoop.cs

```
@@ -6,9 +6,36 @@
 
 namespace UnityEngine.Experimental.ScriptableRenderLoop
 {
+    public class HDRenderLoopInstance : RenderPipeline
+    {
+        private readonly HDRenderLoop m_Owner;
+
+        public HDRenderLoopInstance(HDRenderLoop owner)
+        {
+            m_Owner = owner;
+
+            if (m_Owner != null)
```

is there a valid situation where the owner is null?  if not, it should throw an exception here